framework/audio/iplayer.h

```diff
@@ -0,0 +1,65 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_AUDIO_IPLAYER_H
+#define MU_AUDIO_IPLAYER_H
+
+#include "async/channel.h"
+
+namespace mu {
+namespace audio {
+class IPlayer
+{
+public:
+    enum Status {
+        Stoped = 0,
+        Running,
+        Error
+    };
+
+    virtual ~IPlayer() = default;
+
+    virtual Status status() const { return m_status; }
+    virtual async::Channel<Status> statusChanged() const { return m_statusChanged; }
+
+    virtual bool isRunning() const { return m_status == Status::Running; }
+
+    virtual void run() = 0;
```

Comment:
Better to rename to "play" as was in the previous implementation

Reply:
@RomanPudashkin renamed. Thanks

Reply:
I think it should be renamed back to "run".
Or, we also need to rename one of the status values and the `isRunning` method.
But this "player" is not at all what was previously called a "player".

Reply:
renamed back to "run" (https://github.com/musescore/MuseScore/pull/7274)